components/Storyblok/SbStoryCard.tsx

```diff
@@ -63,9 +66,9 @@ export const SbStoryCard = ({
     heading={heading || cardTitle || title}
     headingLevel={headingLevel}
     isSmallHeading={isSmallHeading}
-    body={dek}
-    imageSrc={cardImage || mobileFilename || heroFilename || bgFilename }
-    imageFocus={cardFocus || mobileFocus || heroFocus || bgFocus}
+    body={cardTeaser || dek}
```

Comment:
Story Card - show card teaser for the card body if there's one enter in the story, otherwise show the story dek